python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -82,14 +88,22 @@ def test_sets(self) -> List[ItemSet] or List[ItemSetDict]:
         """Return the test set."""
         return self._test_sets
 
-    def graph(self) -> object:
+    def graph(self) -> CSCSamplingGraph:
```

Comment:
I think we can still be very soft here to keep it as object.

Reply:
sure as more types might be supported later